src/pages/index.js

```diff
@@ -1,80 +1,165 @@
 import "./index.css";
 import FormValidator from "../components/FormValidator.js";
-import Card from "../components/card.js";
+import Card from "../components/Card.js";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
+import { api } from "../components/Api.js";
 
 import {
-  initialCards,
   validationObj,
   editButton,
   addButton,
+  profileImage,
+  profilePicForm,
   editForm,
   addCardForm,
   modalImage,
   modalTitle,
 } from "../utils/constants.js";
 
-const cardPopup = new PopupWithForm("#add-card-modal", renderNewCard);
-const profilePopup = new PopupWithForm("#edit-profile-modal", editProfile);
-const locationPopup = new PopupWithImage("#location-modal");
+const popups = {
+  cardPopup: new PopupWithForm("#add-card-modal", renderNewCard),
+  profilePopup: new PopupWithForm("#edit-profile-modal", editProfile),
+  deleteCardPopup: new PopupWithForm("#delete-card-modal", deleteCard),
+  profilePicPopup: new PopupWithForm("#profile-pic-modal", editProfilePic),
+  locationPopup: new PopupWithImage("#location-modal"),
+};
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__name",
-  descriptionSelector: ".profile__subtitle",
-});
-const cardSection = new Section(
-  { items: initialCards, renderer: createCard },
-  ".locations"
+const userInfo = new UserInfo(
+  {
+    nameSelector: ".profile__name",
+    descriptionSelector: ".profile__subtitle",
+  },
+  profileImage
 );
+const cardSection = new Section({ renderer: createCard }, ".locations");
 
 // validation class
-const editFormValidation = new FormValidator(validationObj, editForm);
-const cardFormValidation = new FormValidator(validationObj, addCardForm);
+const formValidators = {
+  editFormValidation: new FormValidator(validationObj, editForm),
+  cardFormValidation: new FormValidator(validationObj, addCardForm),
+  picFormValidation: new FormValidator(validationObj, profilePicForm),
+};
 
 // initail setup
-cardPopup.setEventListeners();
-profilePopup.setEventListeners();
-locationPopup.setEventListeners();
+for (let popup in popups) {
+  popups[popup].setEventListeners();
+}
 
-cardSection.renderItems();
+for (let form in formValidators) {
+  formValidators[form].enableValidation();
+}
 
-editFormValidation.enableValidation();
-cardFormValidation.enableValidation();
+api
+  .getAll()
+  .then((data) => {
+    userInfo.setUserInfo(data[0]);
+    userInfo.setUserImage(data[0]);
+    return data;
+  })
+  .then((data) => {
```

Comment:
Needs correction: 

1) there is no need to have two then blocks, you can do everything via one
2) please destructure the data, so it will be clear what it's values contain: `.then(([userData, cardsData]) => { .. })`
3) don't forget to handle the api errors